FreeRTOS/Demo/RISC-V_Galois_P1/link.ld

```diff
@@ -74,14 +74,15 @@ OUTPUT_ARCH( "riscv" )
 
 _STACK_SIZE = DEFINED(_STACK_SIZE) ? _STACK_SIZE : 4K;
 _HEAP_SIZE = DEFINED(_HEAP_SIZE) ? _HEAP_SIZE : 256K;
+_DMEM_LENGTH = DEFINED(_DMEM_LENGTH) ? _DMEM_LENGTH : 0x00800000;

```

Comment:
Is this really configurable? I was under the impression that all CPUs have fixed amount of memory. Right now DMEM is ~8MB, of we have more we should reflect it in this script.

Reply:
I am using these values:
```
_STACK_SIZE = 128K;
_HEAP_SIZE = 8M;
_DMEM_LENGTH = 0x04000000;
```
The dmem overflows if used as 0x00800000 with the above values.

Reply:
Shouldn't the line be then changed to?
```suggestion
_DMEM_LENGTH = DEFINED(_DMEM_LENGTH) ? _DMEM_LENGTH : 0x04000000;
```